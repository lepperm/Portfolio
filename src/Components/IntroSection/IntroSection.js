/**
 * @fileoverview IntroSection component.
 */

import React from "react";
import PropTypes from "prop-types";

/**
 * IntroSection component
 *
 * @return {Component} IntroSection component
 */
export const IntroSection = () => {
  return (
    <section id="introduction">
      <header>
        <h1>Hi, I'm Max!</h1>
        <h2>I'm a software developer focused in React from Dayton, Ohio.</h2>
      </header>
    </section>
  );
};

/**
 * Prop types for IntroSection component
 */
IntroSection.propTypes = {
  //
};
