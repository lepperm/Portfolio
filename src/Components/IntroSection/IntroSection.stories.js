/**
 * @fileoverview Story for IntroSection component.
 */

import React from "react";
import { IntroSection } from "./IntroSection";

export default {
  title: "IntroSection",
  component: IntroSection,
};

export const Basic = () => <IntroSection />;
