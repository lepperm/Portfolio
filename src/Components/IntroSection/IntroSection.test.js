/**
 * @fileoverview Tests for IntroSection component.
 */

import React from "react";
import { shallow } from "enzyme";
import { IntroSection } from "./IntroSection";

describe("IntroSection", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<IntroSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
