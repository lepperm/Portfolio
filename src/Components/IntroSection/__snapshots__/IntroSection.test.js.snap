// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroSection matches snapshot 1`] = `ShallowWrapper {}`;
